Rakefile

```diff
@@ -11,7 +11,8 @@ Vmdb::Application.load_tasks
 begin
   require 'jasmine'
   load 'jasmine/tasks/jasmine.rake'
-rescue LoadError
+rescue LoadError => e
+  puts e.message
```

Comment:
@cynepco3hahue Thanks, it's worthing seeing what other think.  If we want to really do nothing on `LoadError`, it's easy enough to make that explicit.

```ruby
begin
  require 'jasmine'
  load 'jasmine/tasks/jasmine.rake'
rescue LoadError
  # Do nothing
end